src/_imaging.c

```diff
@@ -82,6 +82,12 @@
 #include "zlib.h"
 #endif
 
+#ifdef HAVE_LIBTIFF
+#ifndef _TIFFIO_
+#include <tiffio.h>
```

Comment:
Why is that?

Reply:
Coping the include from TiffDecode.h to bring in `TIFFLIB_VERSION`